ovirt/resource_ovirt_vm.go

```
@@ -168,7 +175,42 @@ func resourceOvirtVMCreate(d *schema.ResourceData, meta interface{}) error {
 	if ok {
 		d.SetId(newVM.MustId())
 	}
-
+	for i := 0; i < numNetworks; i++ {
+		prefix := fmt.Sprintf("network_interface.%d", i)
+		profilesService := conn.SystemService().VnicProfilesService()
+		var profileID, pfDcID, newVMDcID string
+		pfsResp, _ := profilesService.List().Send()
```

`MustSend()` could be used to ignore the error returned.